packages/react/src/components/ExpressiveModal/ExpressiveModal.js

```diff
@@ -50,6 +50,12 @@ const ExpressiveModal = ({
    */
   function closeModal() {
     if (onClose?.() !== false) {
+      if (window.kWidget) {
+        window.kWidget.addReadyCallback(function(playerId) {
+          var kdp = document.getElementById(playerId);
+          kdp.sendNotification('doStop');
+        });
+      }
```

Comment:
Any way we can do this in light box media viewer code instead of backing video player-specific feature in expressive modal?

Reply:
Good point! Sorry I must have misunderstood. Thanks!